static/js/poll_modal.js

```diff
@@ -0,0 +1,76 @@
+import $ from "jquery";
+import {Sortable} from "sortablejs";
+
+import render_poll_modal_option from "../templates/poll_modal_option.hbs";
+
+export function update_option_delete_button_visibility($container, display_flag) {
+    const no_of_option_row = $container.find(".option-row").length;
+
+    // When there is only one option row, we hide its delete button
+    if (no_of_option_row === 1) {
```

Comment:
We don't use abbreviations in variable names; `no_of_option_row` => `option_row_count` would be clearer.